get-started/initialize/step2.md

```diff
@@ -10,7 +10,5 @@
 **Optional:** Install bash completion for `dvc`:
 
 ```
-dvc completion -s bash > /etc/bash_completion.d/dvc
+dvc completion -s bash | sudo tee /etc/bash_completion.d/dvc
```

Comment:
please, make sure that it runs as expected in Katacoda env

Reply:
Done. Tested :+1: